common/static/js/spec/logger_spec.js

```diff
@@ -37,6 +37,74 @@
             });
         });
 
+        describe('ajax request settings with path_prefix', function() {
+            var meta_tag;
+            var step = 0;
+            var meta_tag_name_list = ["path_prefix1", "path_prefix", "path_prefix"];
+            var meta_tag_content_values = ["" , "", "testpath"];
+
+            beforeEach(function(){
+                meta_tag = $('<meta name="'+meta_tag_name_list[step]+'" content="'+meta_tag_content_values[step]+'">');
```

Comment:
don't do this `step` thing, just define a new meta tag in each function. That's much more clear